packages/admin/cms-admin/src/dam/MoveDamItemDialog/useFolderTree.ts

```diff
@@ -91,7 +91,6 @@ export const useFolderTree = ({ damFoldersFlat }: UseFolderTreeProps): UseFolder
         setFoldersToRender(newFoldersToRender);
 
         // This should only be executed if the searchQuery changes
-        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Is this change correct? If yes, we should remove the comment above.

Reply:
removed the comment, as far as i see this change should be correct, all required dependencies are mentioned in the dependency array of the use Effect hook